### DISCUSSION TITLE
`plot_rank`: A New Visualization Tool in Optuna

### DISCUSSION THREAD

**Discussion by contramundum53** [optuna/optuna#4734] [Announcements]
A new visualization function `plot_rank` is introduced in Optuna v3.2. It provides valuable insights into the landscape of high-dimensional objective function. Check out [this blog](https://medium.com/@contramundum53/4b2e858b0920) for more information!

![image](https://miro.medium.com/v2/resize:fit:4800/format:webp/1*1-P-lvw2BnFZAMXn36Ld8g.png)

**Comment by BrunoArsioli** [comment]
Great to have that functionality. It will help visualise the parameter space probed with Trials. I am teaching a ML Module for PhD students in astrophysics, and will test with them. Btw, is there an Optuna page in likedin? It would be nice to engage with the community there, and increase user awareness.

**Reply by contramundum53** [reply]
Thanks for your great suggestion!
We finally have our Page in LinkedIn: https://www.linkedin.com/showcase/optuna
Please follow it if you are interested in updates!
